### PR TITLE
Modify CallStates

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo/issues/2316

### CODE REVIEW

File: src/neo/SmartContract/CallFlags.cs [hunk 1]
```diff
@@ -8,12 +8,11 @@ public enum CallFlags : byte
         None = 0,
 
         ReadStates = 0b00000001,
-        WriteStates = 0b00000010,
+        WriteStates = 0b00000011,
```
Comment (erikzhang) [inline src/neo/SmartContract/CallFlags.cs:11]: I think we'd better not to set two bits in a single flag.

Comment (shargon) [reply src/neo/SmartContract/CallFlags.cs]: I will fix it

Comment (erikzhang) [reply src/neo/SmartContract/CallFlags.cs]: The wallet should always set `ReadStates` together with `WriteStates`.

Comment (Qiao-Jin) [reply src/neo/SmartContract/CallFlags.cs]: How about renaming WriteStates to ReadWriteStates?

Comment (shargon) [reply src/neo/SmartContract/CallFlags.cs]: The problem is if you set `ReadWriteStates` and you require `Read` the flags doesn't match

**Comment by Qiao-Jin** [comment]
Ping

**Comment by erikzhang** [comment]
Please check #2339

**Pull request closed without merging**